src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Consider using 'if' with 'return' instead of 'if else' to simplify the flow of the function. This will make the code cleaner and easier to follow. Also, there's no need for 'else' after an 'if' that contains a 'return'. See checklist items 4 and 5.